_pages/oidc/getting-started.md

```diff
@@ -0,0 +1,86 @@
+---
+title: OpenID Connect
+lead: >
+   [OpenID Connect](http://openid.net){:class="usa-link--external"} is a simple identity layer built on top of the OAuth 2.0 protocol. Login.gov supports [version 1.0](http://openid.net/specs/openid-connect-core-1_0.html){:class="usa-link--external"} of the specification and conforms to the [iGov Profile](https://openid.net/wg/igov){:class="usa-link--external"}.
+redirect_from:
+  - /openid-connect/
+sidenav:
+  - text: Getting started
+    href: "oidc/getting-started/"
+    links:
+      - text: Choosing an authentication method
+        href: "#choosing-an-authentication-method"
+      - text: Set up a Sandbox account
+        href: "#set-up-a-sandbox-account"
+      - text: Auto-discovery
+        href: "#auto-discovery"
+      - text: Example application
+        href: "#example-application"
+  - text: Authorization
+    href: "oidc/authorization/"
+  - text: Token
+    href: "oidc/#token"
+  - text: User info
+    href: "oidc/#user-info"
+  - text: Certificates
+    href: "oidc/#certificates"
+  - text: Logout
+    href: "oidc/#logout"
+  - text: Example application
+    href: "oidc/#example-application"
+
+---
+<div class="usa-alert usa-alert--warning">
+<div class="usa-alert__body">
+<p class="usa-alert__text">We do not support the OpenID Connect “implicit flow” with client_secret because it is not recommended by the OAuth group for security reasons</p>
+</div>
+</div>
```

Comment:
We have the `alert` partial in this project so we ca replace this with
```suggestion
{% capture implicit_flow_warning %}
We do not support the OpenID Connect “implicit flow” with client_secret because it is not recommended by the OAuth group for security reasons
{% endcapture %}
{% include alert.html content=implicit_flow_warning alert_class="usa-alert--warning" %}
```